tests/phpunit/elementor/modules/atomic-widgets/test-atomic-svg.php

```diff
@@ -13,10 +13,10 @@ class Test_Atomic_Svg extends Elementor_Test_Base {
 		'elType' => 'widget',
 		'settings' => [
 			'svg' => [
-				'url' => ELEMENTOR_ASSETS_PATH . '/images/a-default-svg.svg',
+				'url' => ELEMENTOR_ASSETS_PATH . '/images/e-default-svg.svg',
 			],
 		],
-		'widgetType' => 'a-svg',
+		'widgetType' => 'e-svg',
```

Comment:
In all the tests, change to `Svg::get_element_type()` instead of using plain string